### ISSUE TITLE
[BUG] Sudden "Syntax Error" in Userscript

### ISSUE THREAD

**Issue by MarvNC** [violentmonkey/violentmonkey#2133]
TLDR; a userscript I made and have been using for years suddenly has a "syntax error" that prevents it from running.

### Sequence of actions: 
1. Install [this userscript](https://raw.githubusercontent.com/MarvNC/Book-Stats-Charts/main/release-dates.user.js) (code has not been changed in months)
2. Navigate to a page such as https://bookwalker.jp/series/48575/list/
3. Userscript does not run

### Problem: 

![2024-07-01_09-26-26](https://github.com/violentmonkey/violentmonkey/assets/17340496/f82ed64b-6a78-4288-bef2-5a247d366a18)

(tested in beta and stable)

### Expected result:  

There is no syntax error

#### Devtools console contents:

Nothing useful that I could find

#### Environment:
- OS: Windows 10
- Browser: Edge Version 126.0.2592.81 (Official build) (64-bit)
- Violentmonkey Version: 2.19.0, 2.19.3

**Comment by tophf** [comment]
Your script uses https://cdn.jsdelivr.net/npm/chartjs-plugin-trendline which was apparently updated by the author to ES module format and now there's `import {LineFitter} from "./classes/lineFitter.js";` which is invalid outside of a module. Userscripts aren't modules.

Try finding a different distribution for that script in standard format, not a module.

**Comment by tophf** [comment]
Also check the global console Ctrl-Shift-J -> Multiprocess.
